app/models/user.rb

```diff
@@ -11,6 +11,7 @@ class User < ApplicationRecord
   attribute :bio, :text
   attribute :post_counter, :integer, default: 0
   attribute :photo, :string
+  enum role: %i[user admin]
```

Comment:
- [ ] Kindly verify your app functionalities. In your migration file you added a `role` to users table with a column type of `string`, but you are declaring `enum` for it in the model file. In rails, `enum` is typically an integer column type, not a string. I would suggest you stick to the string type you defined. I could not make a user an admin until I removed this line.